### ISSUE TITLE
Adding websocket to other pages

### ISSUE THREAD

**Issue by emladevops** [mely-apps/melyoj#22]

**Comment by nyagami** [comment]
Nếu userbase lớn thì nên cân nhắc, mở websocket quá nhiều có thể gây áp lực tải lớn cho server.
Hoặc tách biệt client vs server, client sẽ là Single Page, như vậy thì chỉ cần khởi tạo 1 connection và tái sử dụng nó

**Comment by emladevops** [comment]
> Nếu userbase lớn thì nên cân nhắc, mở websocket quá nhiều có thể gây áp lực tải lớn cho server.
> 
> Hoặc tách biệt client vs server, client sẽ là Single Page, như vậy thì chỉ cần khởi tạo 1 connection và tái sử dụng nó

Hiện tại với multi connection được build cho websocket, nếu người dùng không có bất kỳ request hay traffic gì đến server sẽ tự động bị disconnect trong 5 phút.

**Comment by nyagami** [comment]
đúng, nhưng người dùng thì họ không đứng yên, việc chuyển trang với SSR sẽ tạo kết nối mới liên tục

**Comment by emladevops** [comment]
probably yes, feature này sẽ được triển khai sau.
